independent-publisher-connectors/Microsoft Partner Center/apiDefinition.swagger.json

```diff
@@ -0,0 +1,895 @@
+                        "title": "Contract Type"
+                      },
+                      "publisherName": {
+                        "type": "string",
+                        "description": "The publisher name.",
+                        "title": "Publisher Name"
+                      },
+                      "promotionId": {
+                        "type": "string",
+                        "description": "The promotion identifier if applied to the subscription.",
+                        "title": "Promotion ID"
+                      },
+                      "orderId": {
+                        "type": "string",
+                        "description": "The ID of the order that was placed to begin the subscription.",
+                        "title": "Order ID"
+                      }
+                    }
+                  },
+                  "description": "items"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get customer subscriptions",
+        "description": "Gets a collection of all the subscriptions for a specified customer.",
+        "operationId": "GetCustomerSubscription",
+        "parameters": [
+          {
+            "name": "CustomerTenantID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Customer tenant ID",
+            "description": "A customer tenant ID",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/v1/customers/{CustomerTenantID}/orders": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "totalCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Total count of customer orders",
+                  "title": "Total Count"
+                },
+                "items": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "An order identifier.",
+                        "title": "ID",
+                        "x-ms-visibility": "important"
+                      },
+                      "alternateId": {
+                        "type": "string",
+                        "description": "alternateId"
+                      },
+                      "referenceCustomerId": {
+                        "type": "string",
+                        "description": "The customer identifier.",
+                        "title": "Customer ID"
+                      },
+                      "billingCycle": {
+                        "type": "string",
+                        "description": "billingCycle"
```

Comment:
@oleksii-innoware All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Hi @sriyen-msft 
Thank you for feedback - I fixed some descriptions. Almost all descriptions I took from MS Docs documentation, but if you find issues - kindly point me as I am not a native speaker and can make mistakes.
Thank you in advance and have a nice day!